WireAPI/Sources/WireAPI/Models/UpdateEvent/ConversationEvent/ConversationMLSMessageAddEvent.swift

```diff
@@ -41,5 +56,14 @@ public struct ConversationMLSMessageAddEvent: Equatable, Codable {
     /// The base 64 encoded message.
 
     public let message: String
+    
+    /// The date the message was received.
+
+    public let timestamp: Date?
+    
+    /// The decrypted current message + decrypted buffered messages
+    /// along with the related sender client ID for each message.
+
+    public var decryptedMessages: [DecryptedMessage] = []
```

Comment:
similar to the Proteus add event with the message content property, this property will be filled in the decryptor when the message + buffered messages are decrypted